doc/classes/VisualScriptReturn.xml

```diff
@@ -1,8 +1,15 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualScriptReturn" inherits="VisualScriptNode" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
+		Exits a function and returns an optional value.
 	</brief_description>
 	<description>
+		This node ends the execution of a function and returns control to the calling function. Optionally, it can return a [Variant] value.
```

Comment:
`Ends the execution of a function and returns control to the calling function. Optionally, it can return a [Variant] value.